app/controllers/api/v0/admin/users_controller.rb

```diff
@@ -0,0 +1,8 @@
+class Api::V0::Admin::UsersController < ApiController
+  before_filter :require_admin
+
+  def index
+    users = User.all.map { |user| UserPresenter.new(user) }
+    render json: AdminReport.new(users: users), serializer: ::Admin::AdminReportSerializer
```

Comment:
interesting  -- why `::Admin::AdminReportSerializer` instead of  `Admin::AdminReportSerializer` ?